### PR TITLE
Add template_fit's get_points function and main function

### PR DESCRIPTION
**Pull request by aspadiyath**

### CODE REVIEW

File: stance/etl/template_fit.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    """
+    COCO Output Format
+    Neck – 1, Right Shoulder – 2, Right Elbow – 3, Right Wrist – 4,
+    Left Shoulder – 5, Left Elbow – 6, Left Wrist – 7, Right Hip – 8,
+    Right Knee – 9, Right Ankle – 10, Left Hip – 11, Left Knee – 12,
+    LAnkle – 13
+
+    Parameters
+    ----------
+    image - image of user
```
Comment (RFinkelberg) [inline stance/etl/template_fit.py]: Format of params and return should be
```
name : type
    description of thing
```

File: stance/etl/template_fit.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    # input image dimensions for the network
+    in_weight = frame.shape[1]
+    in_height = frame.shape[0]
+    inp_blob = cv2.dnn.blobFromImage(frame, 1.0 / 255, (in_weight, in_height),
+                                     (0, 0, 0), swapRB=False, crop=False)
+
+    net.setInput(inp_blob)
+
+    output = net.forward()
+    # print("time taken by network : {:.3f}".format(time.time() - t))
```
Comment (RFinkelberg) [inline stance/etl/template_fit.py]: We might want to turn this into a proper `logging` thing if we end up caring about the timing. If not, we should delete these unused comments

File: stance/main.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+import template_fit
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-f", "--front_view_image", help="image containing the front view of the user")
+    parser.add_argument("-p","--profile_view_image", help="image containing the profile view of the user")
+    parser.add_argument("-v","--motion_video", help="video containing the user performing a motion")
+    args = parser.parse_args()
+
+    template_fit.template_fitting(args.front_view_image, args.profile_view_image)
```
Comment (RFinkelberg) [inline stance/main.py:11]: Does this actually display any images or does it just calculate the points?

**Review by RFinkelberg — APPROVED** [review]
LGTM besides the style stuff

File: stance/main.py (hunk 3 above)
Comment (aspadiyath) [reply stance/main.py:11]: Just the points atm

**Pull request merged**